recipes/sparc-x-api/meta.yaml

```diff
@@ -14,14 +14,13 @@ build:
   noarch: python
   number: 0
   # Install and include the psp files
-  script: "{{ PYTHON }} -m pip install . --no-deps -vv && cd .. && {{ PYTHON }} -m sparc.download_data"
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
   entry_points:
     - "sparc-ase = sparc.cli:main"
 
 requirements:
   host:
     - python >=3.8
-    - ase >=3.22
```

Comment:
Did you try just removing this first? B/c if that works you don't need the post link. Post links will require to remove noarch and that is not desirable.